src/specsynthase/cli.py

```diff
@@ -8,4 +8,4 @@ def main():
     spec = SpecBuilder()
     for f in sys.argv[1:]:
         spec.add_spec(f)
-    print(spec.dump(allow_unicode=True))
+    print(spec.validate().dump(allow_unicode=True))
```

Comment:
The same as connexion, an exception from validator package:

![image](https://user-images.githubusercontent.com/9947646/28996050-80ba1c5e-79ce-11e7-9e2c-41d959ab3970.png)